presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/Matcher.java

```diff
@@ -67,7 +68,8 @@ public interface Matcher
      * @param session The session information for the query
      * @param metadata The metadata for the query
      * @param symbolAliases The SymbolAliases containing aliases from the nodes sources
+     * @param cteNameMapping The one to one mapping between cte name in plan pattern and the cte name got from result query plan
      * @return a MatchResult with information about the success of the match
      */
-    MatchResult detailMatches(PlanNode node, StatsProvider stats, Session session, Metadata metadata, SymbolAliases symbolAliases);
+    MatchResult detailMatches(PlanNode node, StatsProvider stats, Session session, Metadata metadata, SymbolAliases symbolAliases, BiMap<String, String> cteNameMapping);
```

Comment:
Use cteNameMapping to store the one to one mapping of CTE names, so that in the plan test, we do not need to provide the exact CTE name in plan pattern as in result query plan.